docs/design.md

```diff
@@ -101,12 +122,14 @@ Up to `F` failures can be tolerated, where the total number of replicas `N = 2F
 
 # Cockroach Client
 
+[Woah, big update required here!]
 In order to support diverse client usage, Cockroach clients connect to
 any node via HTTPS using protocol buffers or JSON. The connected node
 proxies involved client work including key lookups and write buffering.
 
 # Keys
 
+[Update referencing concepts in keys/constants.go required here]
```

Comment:
I can tackle this section. Will probably rename it to `Internal Keys` to delineate it from `SQL`.